tests/bootstrap.go

```diff
@@ -46,31 +47,57 @@ func truncateAllTablesForTestingOnly() {
 	TruncateTasks := fmt.Sprintf("TRUNCATE TABLE tasks;")
 	TruncateWorkflows := fmt.Sprintf("TRUNCATE TABLE workflows;")
 	TruncateLaunchPlans := fmt.Sprintf("TRUNCATE TABLE launch_plans;")
-	// HACK: alter executions table so that spec is not required for testing.
-	TruncateExecutions := fmt.Sprintf("TRUNCATE TABLE executions; alter table executions alter column spec drop not null;")
+
 	TruncateExecutionEvents := fmt.Sprintf("TRUNCATE TABLE execution_events;")
 	TruncateNamedEntityMetadata := fmt.Sprintf("TRUNCATE TABLE named_entity_metadata;")
-	TruncateNodeExecutions := fmt.Sprintf("TRUNCATE TABLE node_executions;")
 	TruncateNodeExecutionEvents := fmt.Sprintf("TRUNCATE TABLE node_execution_events;")
-	TruncateTaskExecutions := fmt.Sprintf("TRUNCATE TABLE task_executions;")
+	TruncateTaskNodeExecutions := fmt.Sprintf("truncate table task_executions,node_executions,executions;")
```

Comment:
is this change because of foreign key references? can you add a comment explaining why it's necessary?

Reply:
I have removed this change now as we are disabling foreign key constraints during migration which prevents gorm to add these constraints to the tables.